vitest-setup.ts

```diff
@@ -0,0 +1,17 @@
+import { vi } from "vitest";
+
+vi.mock("./src/config", async () => ({
+  AWS_REGION: "ca-central-1",
+  SERVER_PORT: 3001,
+  LOCALSTACK_ENDPOINT: undefined,
+  ZITADEL_APPLICATION_KEY: JSON.stringify({
+    keyId: "test",
+    clientId: "test",
+    key: "test",
+  }),
+  ZITADEL_DOMAIN: "test",
+}));
+
+vi.mock("node:crypto", () => ({
+  createPrivateKey: vi.fn(),
```

Comment:
When the `introspectToken` file is imported in a test file, the private key gets created right away (now that is lives outside of the function).

Because we don't want to have an RSA private key on Github (even if it is not a valid one), we have to mock the `createPrivateKey` function so that it does not try and fail to parse the `ZITADEL_APPLICATION_KEY` that is also mocked just above.